### ISSUE TITLE
better example of stopped callback

### ISSUE THREAD

**Issue by patriknw** [akka/akka#28144]
The example in https://doc.akka.io/docs/akka/current/typed/actor-lifecycle.html#stopping-actors doesn't show a good use of the stopped callback. I think a better usage would be to include a `replyTo` in the message and then send back a reply from the callback. That can't be done with the PostStop signal alone.

Maybe also clarify in reference docs what is mentioned in apidocs:
```
   * The `PostStop` signal that results from stopping this actor will first be passed to the
   * current behavior and then the provided `postStop` callback will be invoked.
   * All other messages and signals will effectively be ignored.
```

**Comment by franciscolopezsancho** [comment]
I tried to do it in https://github.com/franciscolopezsancho/upgrading_to_akka_typed/blob/master/src/main/scala/com/example/Signals.scala

it can be run as a Main. 

You may notice that version is old but that's for a reason.  My first problem here is that callback from 'Behaviors.stopped' goes to dead letters mailbox, but more importantly, @patriknw ,  I found that in version 2.6, with new Logger, signals weren't logged. The very same sample https://github.com/akka/akka/blob/v2.6.1/akka-actor-typed-tests/src/test/scala/docs/akka/typed/GracefulStopDocSpec.scala#L8-L75 doesn't log Signals either.

**Comment by patriknw** [comment]
@franciscolopezsancho I don't think that is a good example, because `Behaviors.stopped` will stop the actor immediately and the `Cleaned` message will never arrive.

I was thinking of adding `replyTo: ActorRef[Done]` to the `GracefulStop` message and then use 
```
  case GracefulStop(replyTo) =>
    // some cleanup
    Behaviors.stopped { () => replyTo ! Done }
```

**Comment by franciscolopezsancho** [comment]
I added an example with a middle man actor (called `Manager`) that will send a message to its supervisor when `Behaviors.stopped` after received a `GracefulShutdown`
I couldn't think of any other possibility than to add a middle man. Otherwise, the sent message would never reach himself if tried in the `MasterControlProgram`
I actually thought about sending a message to an actor of another system but to spin another ActorSystem felt like a bad idea.

**Comment by ennru** [comment]
Fixed with #29760
